README.md

```diff
@@ -65,9 +65,7 @@ let package = Package(
         .package(url: "https://github.com/mongodb/mongo-swift-driver.git", from: "0.0.2"),
     ],
     targets: [
-        Target(
-            name: "MyPackage",
-            dependencies: ["MongoSwift"])
+        .target(name: "MyPackage", dependencies: ["MongoSwift"])
```

Comment:
Obviously, a large app may have many dependencies and use line breaks for declaring the `dependencies` array, but I don't see why we can't use one line for our concise example.

Willing to change if needed.